app/experimenter/experiments/constants.py

```diff
@@ -365,6 +365,7 @@ class EmailType(models.TextChoices):
         Application.IOS: Version.FIREFOX_101,
         Application.FOCUS_IOS: Version.FIREFOX_101,
     }
+    FEATURE_ENABLED_MIN_UNSUPPORTED_VERSION = {Application.DESKTOP: Version.FIREFOX_103}
```

Comment:
Oh i see you wrote it here as 103 and used `>` above.  The change was actually introduced in 104 wasn't it?  Looking at this I would think it was introduced in 103, it would probably be clearer to make it match here the version it was actually changed in.  

Also since we're only doing desktop here, a dictionary is probably unnecessary?

Reply:
makes sense completely :)